tests/phpunit/Integration/MediaWiki/Hooks/ParserFirstCallInitIntegrationTest.php

```diff
@@ -98,7 +98,8 @@ public function testParseWithParserFunctionEnabled( $parserName, $text ) {
 
 		$expectedNullOutputFor = [
 			'concept',
-			'declare'
+			'declare',
+			'ask'
```

Comment:
I dont't think adding ask to "expectedNullOutputFor" is the right way to solve this. I'd expect that "ask" creates some output. This requires further investigation. We could temp. skip this test with a clear message describing the issue.